Can't install keyv-memcache

In package.json, there's an invalid version number:

https://github.com/jaredwray/keyv-memcache/blob/6acfa54fb2a033b84c52a4f00aa5b7d8eea418b0/package.json#L38

This results in an error when downloading `keyv-memcache` from npm:

```
❯ npm i keyv-memcache
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^~> 3.6.5": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/[redacted]/.npm/_logs/2020-09-22T18_43_32_640Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
   },
   "homepage": "https://github.com/jaredwray/keyv-memcache",
   "dependencies": {
-    "codecov": "^~> 3.6.5",
+    "codecov": "^3.6.5",
     "json-buffer": "^3.0.1",
     "memjs": "^1.2.2"
   },

